dcat/rdf/prov.ttl

```diff
@@ -1,56 +1,81 @@
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+dct:publisher
+  rdfs:subPropertyOf prov:wasAttributedTo ;
+.
+dcat:CatalogRecord
+  rdfs:subClassOf prov:Entity ;
+  skos:editorialNote "2017-02-14 SJDC: Propose subClassOf prov:Entity, to enable the use of links to Activities and Agents, for example to satisfy the following requirements https://www.w3.org/TR/dcat-ucr/#RPCX, https://www.w3.org/TR/dcat-ucr/#RFS https://www.w3.org/TR/dcat-ucr/#RPIF " ;
+  skos:editorialNote "2018-02-28 AGB: Links to requirements mentioned above in the github tracker to follow the discussions https://github.com/w3c/dxwg/issues/71, https://github.com/w3c/dxwg/issues/66, https://github.com/w3c/dxwg/issues/76";
+  skos:editorialNote "2017-02-19 SJDC: Alignment axioms put into separate file/graph" ;
+.
+dcat:Dataset
+  rdfs:subClassOf prov:Entity ;
+  skos:editorialNote "2017-01-19 SJDC: Propose subClassOf prov:Entity, to enable the use of links to Activities and Agents, for example to satisfy the following requirements https://www.w3.org/TR/dcat-ucr/#RPCX https://www.w3.org/TR/dcat-ucr/#RFS https://www.w3.org/TR/dcat-ucr/#RPIF " ;
+  skos:editorialNote "2018-02-28 AGB: Links to requirements mentioned above in the github tracker to follow the discussions https://github.com/w3c/dxwg/issues/71, https://github.com/w3c/dxwg/issues/66, https://github.com/w3c/dxwg/issues/76" ;
+  skos:editorialNote "2017-02-19 SJDC: ALignment axioms put into separate file/graph" ;
+.
+<http://www.w3.org/ns/dcat/prov>
+  rdf:type owl:Ontology ;
+  dct:created "2018-02-19"^^xsd:date ;
+  dct:creator <http://registry.it.csiro.au/sandbox/csiro/person/Simon-Cox> ;
+  dct:creator <https://www.w3.org/2017/dxwg/wiki/Main_Page> ;
+  dct:contributor <http://orcid.org/0000-0002-8742-7730> ;
+  dc:contributor "Nicholas Car" ;
+  rdfs:comment "NOTE: this alignment is provisional and incomplete, and is added to the repository here to (a) support further development of the DCAT-PROV alignment (b) demonstrate the strategy of modularizing alignments and dependencies into separate RDF files." ;
+  rdfs:comment "RDF graph containing alignment axioms between DCAT and PROV-O" ;
+  rdfs:label "DCAT-PROV alignment" ;
+  rdfs:seeAlso <https://github.com/w3c/dxwg/issues/134> ;
+  rdfs:seeAlso <https://github.com/w3c/dxwg/issues/111> ;
+  rdfs:seeAlso <https://github.com/w3c/dxwg/issues/128> ;
+  rdfs:seeAlso <https://github.com/w3c/dxwg/pull/94> ;
+  owl:imports <http://www.w3.org/ns/dcat> ;
+  owl:imports <http://www.w3.org/ns/prov-o> ;
+.
+prov:Agent
+  rdfs:comment "prov:Agent is modeled as a subclass of foaf:Agent since it is the subset of agents that are involved in prov:Activity individuals" ;
+  rdfs:subClassOf foaf:Agent ;
+.
+<https://www.w3.org/2017/dxwg/wiki/Main_Page>
+  rdf:type foaf:Agent ;
+  rdfs:label "W3C Data Exchange Working Group" ;
+.
+
+dcmitype:Dataset
+  rdfs:subClassOf prov:Entity ;
```

Comment:
I don't think it's kosher to make this assertion about a resource outside of the control of the WG.